app/controllers/dids_controller.rb

```diff
@@ -5,6 +5,10 @@ class DidsController < DashboardController
   # cache selectable pools before update
   before_action :assign_params, :capacity_pools_for_did, only: [:update]
 
+  before_action only: [:show] do
+    @address_verification = AddressVerificationDecorator.decorate(resource.address_verification)
```

Comment:
it should be done in a different way

on `DidDecorator` use 
```
decorate_association :address_verification, with: AddressVerificationDecorator
```

and access verification through `resource.address_verification` instead of additional instance variable